### PR TITLE
(FACT-1753) Update sles fact acceptance test for SLES 12 Power8

### PR DESCRIPTION
**Pull request by ScottGarman**
This allows me to run the facter acceptance tests with a SLES 12 Power8 target.

### CODE REVIEW

File: acceptance/tests/facts/sles.rb [hunk 1]
```diff
@@ -83,6 +87,7 @@
         kernel_version = '3.0'
       elsif os_version == '12'
         kernel_version = '3.12'
+        kernel_version = '4.4' if os_arch == 'ppc64le'
```
Comment (branan) [inline acceptance/tests/facts/sles.rb]: What's up with this wildly different kernel version?

Comment (ScottGarman) [reply acceptance/tests/facts/sles.rb]: The Power8 hosts uses SLES 12 SP2, whereas our x86 ones are based on SP1.

Comment (branan) [reply acceptance/tests/facts/sles.rb]: That is a hilarious kernel bump for a service pack O_O

**Comment by ericldelaney** [comment]
👍

**Pull request merged**